test/service/test_manifest.py

```diff
@@ -1379,36 +1379,20 @@ def test_manifest_content_disposition_header(self):
                         self.assertEqual(expected_cd, actual_cd)
 
     def test_verbatim_jsonl_manifest(self):
-        expected = []
-        for bundle in self.bundles():
-            bundle = self._load_canned_bundle(bundle)
-            expected.append({
-                'type': 'links',
-                'value': bundle.links
-            })
-            for ref in [
-                'cell_suspension/412898c5-5b9b-4907-b07c-e9b89666e204',
-                'project/e8642221-4c2c-4fd7-b926-a68bce363c88',
-                'sequence_file/70d1af4a-82c8-478a-8960-e9028b3616ca',
-                'sequence_file/0c5ac7c0-817e-40d4-b1b1-34c3d5cfecdb',
-                'specimen_from_organism/a21dc760-a500-4236-bcff-da34a0e873d2',
-                'donor_organism/7b07b9d0-cc0e-4098-9f64-f4a569f7d746',
-                'library_preparation_protocol/9c32cf70-3ed7-4720-badc-5ee71e8a38af',
-                'sequencing_protocol/61e629ed-0135-4492-ac8a-5c4ab3ccca8a',
-                'process/771ddaf6-3a4f-4314-97fe-6294ff8e25a4'
-            ]:
-                expected.append({
-                    'type': EntityReference.parse(ref).entity_type,
-                    'value': bundle.metadata[ref],
-                })
-
         response = self._get_manifest(ManifestFormat.verbatim_jsonl, {})
         self.assertEqual(200, response.status_code)
+        # The canned manifest is saved as a json array instead of jsonl so that
```

Comment:
JSON and JSON Lines are officially the names of the formats. `.json` and `.jsonl` are the respective file extensions. I think this comment would be better placed in `_assert_jsonl` which I think should go into `CannedManifestTestCase`.